packages/foam-core/src/model/workspace.ts

```diff
@@ -56,7 +56,7 @@ export class FoamWorkspace implements IDisposable {
   set(resource: Resource) {
     const id = uriToResourceId(resource.uri);
     const old = this.find(resource.uri);
-    const name = uriToResourceName(resource.uri);
+    const name = uriToResourceName(resource.uri).toLowerCase();
```

Comment:
this is connected to my comment in the previous PR about needing to iron out the mapping of resources and wikilinks. Specifically, that means that `pathToResourceId` and `pathToResourceName`.

In that spirit, have you considered having `toLowerCase` in those functions to "normalize" the paths?

It is true that there might be case conflicts (in file path), but that seems like such an edge case I am ok not optimizing for it.

Regarding your comment about `[[Page]]` and `[[page]]`, the expected behavior would need to be properly addressed (my guess is that in the future `/path/to/Page.md` and `/another/page.md` will be treated just like the case of `/path/to/page.md` and `/another/page.md`). This needs a proper spec to be written though, for which this PR and your previous one already provide two cases.

Reply:
we also talked about this here: https://github.com/foambubble/foam/pull/666#discussion_r649152167

Reply:
> In that spirit, have you considered having toLowerCase in those functions to "normalize" the paths?

I am still wrapping my head around how everything works, so went this route to minimise impact and unintended side effects. But can have a look if things will work as it should when I put in the utility functions.

Reply:
ok, I will wait to hear about the investigation before moving forward with the PR. Let me know if you have any question or need support. If the rabbit hole goes too deep, we can re-evaluate whether to simply go with this interim solution

Reply:
Seems to don't cause any side effect, so comfortable making the change.